zerver/context_processors.py

```diff
@@ -56,19 +56,19 @@ def zulip_default_context(request):
     """
     realm = get_realm_from_request(request)
 
+    realm_uri = settings.SERVER_URI
+    realm_name = None
+    realm_icon = None
+    realm_description = None
+    realm_invite_required = False
```

Comment:
I was following this approach after raising it in #5730 and receiving this comment from @timabbott : https://github.com/zulip/zulip/pull/5730#issuecomment-320053236